orcid-model/src/main/resources/record_2.0_rc2/activities-2.0_rc2.xsd

```diff
@@ -65,7 +65,7 @@
 					</xs:annotation>
 					<xs:complexType>
 						<xs:sequence>
-							<xs:element ref="common:last-modified-date" minOccurs="1" maxOccurs="1" />
+							<xs:element ref="common:last-modified-date" minOccurs="0" maxOccurs="1" />
```

Comment:
@amontenegro i removed minOccurs=1 and added minOccurs=0, so it is not mandatory..